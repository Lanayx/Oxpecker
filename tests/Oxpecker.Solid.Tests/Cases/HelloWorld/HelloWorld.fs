module Oxpecker.Solid.Tests.Cases.HelloWorld

open Browser
open Oxpecker.Solid

[<SolidComponent>]
let Test () =
    console.log "Hello, World!"
    body()
