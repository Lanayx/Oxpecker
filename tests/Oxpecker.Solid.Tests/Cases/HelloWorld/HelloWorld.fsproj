<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Fable.Browser.Dom" Version="2.17.0" />
    <PackageReference Include="Fable.Core" Version="4.3.0" />
  </ItemGroup>

  <ItemGroup>
    <Compile Include="HelloWorld.fs" />
  </ItemGroup>

    <ItemGroup>
        <ProjectReference Include="..\..\..\..\src\Oxpecker.Solid\Oxpecker.Solid.fsproj" />
    </ItemGroup>

</Project>
