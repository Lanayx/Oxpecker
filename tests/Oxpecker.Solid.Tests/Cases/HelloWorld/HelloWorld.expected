import { some } from "./fable_modules/fable-library-js.4.21.0/Option.js";

export function Test() {
    console.log(some("Hello, World!"));
    return <body></body>;
}

